tests/testthat/test-lmer_multimember.R

```diff
@@ -272,3 +272,27 @@ test_that("calling lmer with lmerTest loaded returns the correct types", {
   expect_s3_class(summ, "summary.merMod")
   expect_s3_class(summ, "summary.lmerModLmerTestMultiMember")
 })
+
+test_that("bootstrap and profile likelihood CIs work (and roughly match)", {
+  # make some toy data
+  x <- runif(600, 0, 1)
+  df <- data.frame(
+    x = x,
+    y = rep(rbinom(12, 1, 0.6), 50) + (x * 1.0),
```

Comment:
why `rbinom`?

Reply:
Oh, because commenting the `+ (x + 1.0)` bit and adding a `g` in front of `lmer` let me quickly test code for glmer as well. It does look silly now, haha.